opennlp-tools/src/main/java/opennlp/tools/postag/POSModel.java

```diff
@@ -177,24 +180,10 @@ protected boolean skipEntryForSerialization(Map.Entry<String, Object> entry) {
     return GENERATOR_DESCRIPTOR_ENTRY_NAME.equals(entry.getKey()) && entry.getValue() == null;
   }
 
-  /**
-   * @deprecated use {@link POSModel#getPosSequenceModel} instead. This method will be removed soon.
-   * Only required for Parser 1.5.x backward compatibility. Newer models don't need this anymore.
-   */
-  @Deprecated
-  public MaxentModel getPosModel() {
-    if (artifactMap.get(POS_MODEL_ENTRY_NAME) instanceof MaxentModel) {
-      return (MaxentModel) artifactMap.get(POS_MODEL_ENTRY_NAME);
-    }
-    else {
-      return null;
-    }
-  }
```

Comment:
Maybe adding it to the list of changes in the next release, as it's a public method.

Reply:
Started https://ci-builds.apache.org/job/OpenNLP/job/eval-tests-configurable/19/ to see, if the eval tests rely on older models (as mentioned in the comment)